extensions/amp-list/0.1/amp-list.js

```diff
@@ -237,7 +240,12 @@ export class AmpList extends AMP.BaseElement {
    * @private
    */
   fetch_(itemsExpr) {
-    return fetchBatchedJsonFor(this.getAmpDoc(), this.element, itemsExpr);
+    // Require opt-in for URL variable replacements on fetches triggered
+    // by [src] mutation. @see spec/amp-var-substitutions.md
+    const policy = (this.element.getAttribute('src') == this.initialSrc_)
```

Comment:
Can we just compare the origin's and allow path and hash to differ? That would make it a less of a breaking change and I don't see particular security concerns if origins match, but I might be wrong.